prow/config.yaml

```diff
@@ -19637,15 +19642,19 @@ periodics:
         value: /etc/ssh-key-secret/ssh-public
       - name: USER
         value: prow
-      - name: KUBEMARK_BAZEL_BUILD
-        value: "y"
       volumeMounts:
       - name: service
         mountPath: /etc/service-account
         readOnly: true
       - name: ssh
         mountPath: /etc/ssh-key-secret
         readOnly: true
+      ports:
+      - containerPort: 9999
```

Comment:
This doesn't seem to me like docker-in-docker. This is the docker-out-of-docker approach, as you're using the host node's docker daemon instead of creating a nested docker instance inside the pod.

See - https://applatix.com/case-docker-docker-kubernetes-part-2/

Reply:
No, we're using docker inside of docker. We do not touch the host docker.

Reply:
https://docs.google.com/document/d/1ih0Pogkdyo8QwmJUkJd4pgqmXnPb6vZnUEyb8vhMjJg/edit#
https://github.com/kubernetes/test-infra/tree/master/images/bootstrap

Reply:
I see.. I fail to see where we are starting the docker instance inside the pod though.

Reply:
setting `env DOCKER_IN_DOCKER_ENABLED=true` on the image flips a switch in the runner
https://github.com/kubernetes/test-infra/blob/master/images/bootstrap/runner

Reply:
Aah.. got it. Thanks!